packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -190,12 +189,31 @@ export class GumbyController {
     }
 
     private async transformInitiated(message: any) {
-        // feature flag for SQL transformations
-        if (!isSQLTransformReady) {
+        // silently check for projects eligible for SQL conversion
+        let embeddedSQLProjects: TransformationCandidateProject[] = []
+        try {
+            embeddedSQLProjects = await getValidSQLConversionCandidateProjects()
+        } catch (err) {
+            getLogger().error(`CodeTransformation: error validating SQL conversion projects: ${err}`)
```

Comment:
Just for reference: `getLogger()` now supports a topic, so you can declare a shared logger at module scope:

    const logger = getLogger('CodeTransform')

then use `logger` anywhere in the module. And the topic will prefix each message.

Reply:
Good to know; will update all of our logging with this in my next PR